docs/slack.rst

```diff
@@ -27,16 +27,26 @@ Feel free to send a Pull Request to update `this page`_, if you want to include
 Slack History
 -------------
 
-We don't pay for our Slack, which means that the community has access to only the most
-recent 10,000 messages. Paying for Slack is prohibitively expensive for a community
-like ours, but many people seem to feel a freedom in knowing that their words will disappear shortly.
-We understand that not everyone loves this decision,
-but it's the way things are currently set up and we don't plan to change it in the near future. 
+We don't pay for our Slack, which means that the community has access to only the most recent 10,000 messages.
+That doesn't mean everything gets lost, though.
+Here are a few ways to preserve what you've learned for yourself and others:
 
-That doesn't mean everything gets lost, though - you can always react to a message with the 
-``:suggest-for-newsletter:`` emoji to nominate it for inclusion in our next monthly newsletter. 
-Read the newsletter archives at :doc:`/newsletter` and subscribe if you want them directly in your inbox! 
+- React to a message with ``:suggest-for-newsletter:`` emoji to nominate it for inclusion in our next monthly newsletter.
+  Read the newsletter archives at :doc:`/newsletter/` and subscribe if you want them directly in your inbox!
+- Contribute a summary to one of the Write the Docs site's :doc:`/about/learning-resources`.
+- Turn a discussion into a talk proposal for :doc:`an upcoming Write the Docs conference </conf/index>`.
+- `Save a message`__ privately in Slack.
+- Recap what you learned and share it on your own site or social media accounts. Don't forget to tag `#writethedocs`!
 
+.. __: https://slack.com/intl/en-gb/help/articles/360042650274-Save-messages-and-files-
+
+.. note::
+
+   Why don't we pay for Slack?
```

Comment:
```suggestion
.. admonition:: Why don't we pay for Slack?
   :class: note

```

Unless I'm completely wrong or WtD's theme doesn't work this way, I *think* this should create a note admonition with "Why don't we pay for Slack" as part of the bolded card title.

Reply:
:100:

Reply:
I tried this first and it inserts `:` at the end of the title 😞

Reply:
weird. I didn't even notice that. :/

I keep trying to use them in the conf pages well, :facepalm:

Reply:
I guess cos `Note:` does too

Reply:
Could reword to 

```
.. admonition:: Why we don't pay for Slack
```